test/moonwall.config.json

```diff
@@ -907,6 +907,7 @@
             "timeout": 600000,
             "testFileDir": ["suites/zombie_tanssi_relay"],
             "runScripts": [
+                "test-build-spec.sh",
```

Comment:
Can you move this test to another suite, maybe one that we run weekly? I don't want this one to take longer to start.

Reply:
sure